### ISSUE TITLE
Install fails on python 3.9

### ISSUE THREAD

**Issue by phillipdupuis** [phillipdupuis/dtale-desktop#5]
As noted by a user: 

It fails on Python 3.9 though because of the llvmlite dependency

(https://github.com/numba/numba/issues/6345)

**Comment by joshjimenez0** [comment]
Any way around this for Python 2?

**Comment by phillipdupuis** [comment]
> Any way around this for Python 2?

I don't think so, sadly - fastapi and pydantic both require 3.6+. You could swap in flask for the fastapi part, but replacing the pydantic models would be a bit trickier

It depends, though...do you need the whole app to be running in python 2, or do you just want it to be able to execute python 2 code for loading data?
